media_driver/agnostic/gen12/hw/vdbox/mhw_vdbox_hcp_g12_X.cpp

```diff
@@ -3301,6 +3301,11 @@ MOS_STATUS MhwVdboxHcpInterfaceG12::AddHcpDecodeSliceStateCmd(
             cmd->DW2.NextslicestartctbyOrNextSliceStartLcuYEncoder = hevcSliceStateG12->u16NextTileCtbY;
         }
     }
+    else if (hevcSliceStateG12->dwSliceIndex == 0)
```

Comment:
#Line125 and #Line144 have a check for segment addr for first slice, this error should be found there. Could you help confirm it?